<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>361</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Conectare</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>66</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>IP Server</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>input_ip</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>66</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>input_port</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_username">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>admin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_password">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>admin</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_connect">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>93</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>CONNECT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_register">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>93</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>REGISTER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_heartbeat">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>93</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>HEARTBEAT</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>120</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="log_console">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>361</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
